webboot/webboot.go

```diff
@@ -23,9 +40,88 @@ var (
 	ipv6    = flag.Bool("ipv6", true, "use IPV6")
 )
 
+func wgetArgs(argURL string) (read io.ReadCloser, outFile string, werr error) {
```

Comment:
put a comment here. It's not clear what all this code is doing now.
// getURL takes a URL and returns an io.Reader, URL, and error if any. 
// The io.Reader is returned from wget. 
// If the URL is found in osSystems, the DownloadLink specified there is used and
// returned instead of the passed-in URL. 

Something like that but use your own words.

Reply:
I see what you mean here. For a reader, the implementations of this function might need some explaining. I provided a good description of what the function is exactly doing. There are even comments above certain lines of code within this function that might need more explaining involved. 

I did tweak the comments slightly because the function does not necessarily take in an URL all the time. It is simply an argument that user passes in whether that is a URL or name of ISO that can be found within the osSystems map.

Reply:
I see what you mean here. For a reader, the implementations of this function might need some explaining. I provided a good description of what the function is exactly doing. There are even comments above certain lines of code within this function that might need more explaining involved.

I did tweak the comments slightly because the function does not necessarily take in an URL all the time. It is an argument that user passes in whether that is a URL or name of ISO that can be found within the osSystems map.